### PR TITLE
Upmerge 1.6 to 1.7 20220124

### PR DESCRIPTION
**Pull request by greenie-msft**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Upmerge 1.6 to 1.7 20220124

### CODE REVIEW

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 1]
```diff
@@ -41,13 +41,14 @@ The table below shows the versions of Dapr releases that have been tested togeth
...
+| Sep 15th 2021 | 1.4</br>   | 1.4.0 | Java 1.3.0 </br>Go 1.2.0 </br>PHP 1.1.0 </br>Python 1.3.0 </br>.NET 1.4.0 | 0.8.0 | Unsupported           |
+| Sep 22nd 2021 | 1.4.1</br>   | 1.4.0 | Java 1.3.0 </br>Go 1.2.0 </br>PHP 1.1.0 </br>Python 1.3.0 </br>.NET 1.4.0 | 0.8.0 | Unsupported
+| Sep 24th 2021 | 1.4.2</br>   | 1.4.0 | Java 1.3.0 </br>Go 1.2.0 </br>PHP 1.1.0 </br>Python 1.3.0 </br>.NET 1.4.0 | 0.8.0 | Unsupported           |
+| Oct 7th 2021  | 1.4.3</br>   | 1.4.0 | Java 1.3.0 </br>Go 1.2.0 </br>PHP 1.1.0 </br>Python 1.3.0 </br>.NET 1.4.0 | 0.8.0 | Unsupported |
+| Dev 6th 2021  | 1.4.4</br>   | 1.4.0 | Java 1.3.0 </br>Go 1.2.0 </br>PHP 1.1.0 </br>Python 1.3.0 </br>.NET 1.4.0 | 0.8.0 | Unsupported |
+| Nov 11th 2021 | 1.5.0</br>   | 1.5.0 | Java 1.3.0 </br>Go 1.3.0 </br>PHP 1.1.0 </br>Python 1.4.0 </br>.NET 1.5.0 </br>JS 1.0.2 | 0.9.0 | Supported |
+| Dec 6th 2021  | 1.5.1</br>   | 1.5.1 | Java 1.3.0 </br>Go 1.3.0 </br>PHP 1.1.0 </br>Python 1.4.0 </br>.NET 1.5.0 </br>JS 1.0.2 | 0.9.0 | Supported |
+| Jan 24th 2022 | 1.6.0</br>   | 1.6.0 | Java 1.4.0 </br>Go 1.4.0 </br>PHP 1.1.0 </br>Python 1.5.0 </br>.NET 1.6.0 </br>JS 1.1.0 | 0.9.0 | Supported (current) |
```
Comment (paulyuk) [inline daprdocs/content/en/operations/support/support-release-policy.md]: These look right in theory once builds with tag/release is published.  Let's make sure these are produced. I cannot verify JS and Go currently.  Also why is Python and Java missing releases, whilst having new Tags?

**Review by paulyuk — APPROVED** [review]
Please see q.  It does not block approval, but good to verify.

File: daprdocs/config.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Site Configuration
-baseURL = "https://v1-6.docs.dapr.io/"
+baseURL = "https://docs.dapr.io/"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]: This should be v1-7.

File: daprdocs/config.toml [hunk 3]
```diff
@@ -163,17 +163,20 @@ github_subdir = "daprdocs"
 github_branch = "v1.6"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]: v1.7

**Review by AaronCrawfis — APPROVED** [review]
config.toml needs to be updated. Can either be updated here or in another PR

**Review by msfussell — APPROVED** [review]
LGTM

File: .github/workflows/website-v1-7.yml [hunk 4]
```diff
@@ -3,11 +3,11 @@ name: Azure Static Web App v1.6
 on:
```
Comment (willtsai) [inline .github/workflows/website-v1-7.yml:3]: won't let me comment on line #1, but should update the name to v1.7

**Review by msfussell — APPROVED** [review]
LGTM

**Review by willtsai — APPROVED** [review]

**Pull request merged**